clientcontroller/factory.go

```diff
@@ -13,9 +13,9 @@ import (
 )
 
 const (
-	BabylonConsumerChainName   = "babylon"
-	OPStackL2ConsumerChainName = "OPStackL2"
-	WasmConsumerChainName      = "wasm"
+	BabylonConsumerChainType   = "babylon"
+	OPStackL2ConsumerChainType = "OPStackL2"
+	WasmConsumerChainType      = "wasm"
```

Comment:
Why not use an enum / constants here?

Reply:
what do you mean? those enums are already used in code

Reply:
Oh, I see. Those _already are_ enums in Go.

Perhaps matching against the const values instead of the strings then?